### ISSUE TITLE
Save attendee data once Pay Now is clicked

### ISSUE THREAD

**Issue by SaptakS** [fossasia/open-event-orga-server#2936]
Right now the attendee data gets saved only after the order has been placed or completed. What we want to achieve is save the attendee data immediately after pay now is clicked with status="pending".

**Comment by shubham-padia** [comment]
@SaptakS @mariobehling I followed the discussion on #2934.

Some points i'd like to mention.
- Currently the data is indeed being saved on clicking pay now.
- In the discussion there seemed to be a confusion b/w `Initialized` and `Pending` status 
  - `Initialized` - User has entered the attendee details and clicked on pay now.
  - `Pending` -  User is on the fill attendee information page and has not clicked on Pay Now

- After clicking pay now, order status is changed to `Initialized`, so `pending` status tab seems redundant and instead an `Initialized` tab should be added
- In addition to that an `Initialized` tab should also been added to orders section.

If the above seems okay, i'd like to work on that.

**Comment by mariobehling** [comment]
Please do not add an initialized status. This is information is not relevant for organizers. Please follow existing services. Maybe we should postpone this as well, cause there are bugs that really prevent us from a release whereas the initialize information add little value for the actual event management. Please help to keep things simple. We do not need to re-invent things. Just follow existing services.

**Comment by shubham-padia** [comment]
@mariobehling I think there is a misunderstanding, there is no mention of re-inventing. The `Initialized` status is a pre-existing field already existing in the current app

> - Currently the data is indeed being saved on clicking pay now.
> - In the discussion there seemed to be a confusion b/w `Initialized` and `Pending` status 
>    - `Initialized` - User has entered the attendee details and clicked on pay now.
>   - `Pending` -  User is on the fill attendee information page and has not clicked on Pay Now
>  - In attendees section, After clicking pay now, order status is changed to `Initialized`

 The above comment clarifies how things are working with the app right now.

**Comment by mariobehling** [comment]
Please keep this discussion shorter: No additional UI statuses. Keep it simple for users.

**Comment by shubham-padia** [comment]
> No additional UI statuses

The initialized status is not additional, it is already displayed in the UI, then should it be removed?

> Please keep this discussion shorter

Agreed. Maybe we can come on this later.

**Comment by SaptakS** [comment]
@shubham-padia I have already fixed this. System crashed yesterday so couldn't send a PR.
